aldryn_events/utils.py

```diff
@@ -61,7 +61,7 @@ def build_events_by_year(events, **config):
         if year not in events_by_year:
             events_by_year[year] = {
                 'year': year,
-                'date': date(year, 1, 1),
+                'date': datetime.date(year, 1, 1),
```

Comment:
notice that a lot of calls to datetime.date() can accumulate, best to do something like ``date = datetime.date`` in the local scope of the function.